src/web/components/Picture.tsx

```diff
@@ -27,25 +29,53 @@ const getClosestSetForWidth = (
     });
 };
 
-const getFallbackSrc = (srcSets: SrcSetItem[]): string => {
+const getSourcesForRole = (imageSources: ImageSource[], role: RoleType) =>
+    imageSources.filter(
+        ({ weighting }) =>
+            // Use toLowerCase to handle cases where we have halfWidth comparing to halfwidth
+            weighting.toLowerCase() === role.toLowerCase(),
```

Comment:
Side note: I previously read `toUpperCase` was faster, but it doesn’t seem to hold true in testing.

Reply:
Interesting. I've always defaulted to lowercase in these situations - I guess just because most text is lowercase (unless you work in the media team for a certain american politician) but there's obviously no difference so I'm happy to change it

Reply:
If anything a `toLowerCase()` tends to perform better on [jsbench.me](https://jsbench.me/6pkikaign0/1)

![jsbench lower upper test comparison](https://user-images.githubusercontent.com/76776/101907334-67eb5880-3b88-11eb-883d-b98a2e494063.png)